samples/snippets/README.rst

```diff
@@ -266,6 +266,48 @@ To run this sample:
 
 
 
+Schema
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com//googleapis/python-pubsub&page=editor&open_in_editor=samples/snippets/iam.py,samples/snippets/README.rst
```

Comment:
```suggestion
   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/python-pubsub&page=editor&open_in_editor=samples/snippets/iam.py,samples/snippets/README.rst
```

Reply:
As is, the button triggers a warning that say "this repo isn't maintained by google, do you trust it, blah blah" and then when I said trust it went to the last repo I'd cloned in cloud shell, which was from another button

Additionally, it looks like this button opens up the `README` and `snippets/iam` -is that intended or is it a copy paste error?

Reply:
>  it went to the last repo I'd cloned in cloud shell
Not bad actually. It goes to the current repo for me despite having other repos in the root directory. 

Copy-paste error corrected.